packages/balancer-gql/src/queries/pools.ts

```diff
@@ -14,3 +14,9 @@ export const poolWhereOwner = gql`
     }
   }
 `;
+
+export const poolExists = gql`
+  query PoolExists($poolAddress: ID!) {
+    pool(id: $poolAddress) { symbol }
+  }
+`
```

Comment:
imo this name gives the impression you'll get a boolean in return, which is incorrect. Would rename it to something more accurate like `Pool` and then use the generated `usePool` hook